### ISSUE TITLE
Has the code been reproduced

### ISSUE THREAD

**Issue by ST0427** [lunan0320/Federated-Learning-Knowledge-Distillation#2]
Dear friend,
      Hello friends, have you reproduced this code, why do I reproduce the code accuracy at the highest time is only 77, and there is another problem, reproduce --dataset SVHN --batch_size 64 --num_epochs 50 --clip_grad 1.1 --lr 0.001 --num_clients 10 --num_classes 10 --sampling_rate 1 --local_ ep 5 --beta 0.5 --seed 0 --code_len 32 --alg FedHKD --part 0.1 --model shufflenet --temp 0.5 This parameter, why does loss2:Inf always appear? The -lr_sh_rate parameter he gave doesn't seem to be used. If you reproduce it, can you tell me the parameters you use, and this loss2:inf problem, if you know how to solve it, can you tell me?
      Looking forward to hearing from you.

**Comment by lunan0320** [comment]
First, thank you for your issue. This code is reproduced and modified from the original paper. I proposed a modification plan to the original code and published it in the issue of the original repo, and the author accepted it. The code has been reproduced, but the accuracy may not be that high. I have only tested it on the CIFAR10 dataset. The parameters are in the option.py file. The SVHN data set has not been tested yet.
